torchci/pages/flakytest.tsx

```diff
@@ -0,0 +1,77 @@
+import { FlakyTestData } from "lib/types";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import styles from "components/flakytest.module.css";
+
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
+
+function convertWorkflowIDtoURLs(workflowIds: string[]): string[] {
+  return workflowIds.map((element) => {
+    return `https://github.com/pytorch/pytorch/actions/runs/${element}`;
+  });
+}
+
+export default function Page() {
+  const router = useRouter();
+  const name = (router.query.name || "%") as string;
+  const suite = (router.query.suite || "%") as string;
+  const file = (router.query.file || "%") as string;
```

Comment:
make sure we want to convert "" to "%" as well. Otherwise if we're just doing null checks, prefer ??

Reply:
https://hud2.pytorch.org/flakytest?name=test_success&suite=&file=

It seems to do that already